tools/build/webcomponent/prepare-wc-pages.sh

```diff
@@ -4,7 +4,7 @@ DIST_DEMO_PATH=dist/demo/
 DIST_WC_PATH=${DIST_DEMO_PATH}webcomponents/
 
 echo '-- Build Geonetwork Web Components'
-nx run webcomponents:build --output-hashing=none --output-path=${DIST_WC_PATH}
+nx run webcomponents:build --output-hashing=none --skip-nx-cache --output-path=${DIST_WC_PATH}
```

Comment:
it's more secure if we wan't to build prod artifact to use no cache, isn't it ? I added it because you had issue with an empty `gn-wc.js`, which happened to me as well from time to time.

Reply:
yep, good point